src/main/VideoPlayers.tsx

```diff
@@ -37,7 +37,7 @@ const VideoPlayers: React.FC<{refs: any, widthInPercent?: number}> = ({refs, wid
     borderRadius: '5px',
     ...(flexGapReplacementStyle(10, false)),
 
-    maxHeight: '300px',
+    maxHeight: maxHeightInPixel + 'px',
```

Comment:
Editor is also using emotion-js, right? You should be able to just pass integers here. Also the `borderRadius: 5` above should just work. But if the whole codebase uses `+ 'px'` already, then it's probably not necessary to fix this for this one PR now.

Reply:
It does use Emotion, but not emotion-js specifically I think? Anyway, I like being specific about my units :D